<details><summary>Failed to submit graph for 2d19023eb8640c9d1b2aeea354175be50307c27f
</summary>Message:
Failed to run command: 'pbp graph create --host https://package-build.internal.anaconda.com/pkg-orchestrator -f /tmp/tmp27xzp82z/libcxx-feedstock.yaml --with-linter', with error: 'INFO:root:Loading Graph Data from /tmp/tmp27xzp82z/libcxx-feedstock.yaml
INFO:pbpcli:Creating Graph with group name: libcxx-feedstock/pull/22/commits/2d19023
INFO:root:b'{"detail":[{"type":"value_error","loc":["body","tasks",0,"build","build_options"],"msg":"Value error, Invalid build_options: [\'-c https://staging.continuum.io/prefect/llvm-21.1.8-stage2-build-0\']. Allowed: [\'--debug\', \'--error-overdepending\', \'--error-overlinking\', \'--no-error-overlinking\', \'--no-include-recipe\', \'--no-test\', \'--no-verify\', \'--quiet\', \'--skip-existing\', \'--suppress-variables\']","input":["--suppress-variables","--error-overlinking","-c https://staging.continuum.io/prefect/llvm-21.1.8-stage2-build-0"],"ctx":{"error":{}}},{"type":"value_error","loc":["body","tasks",1,"build","build_options"],"msg":"Value error, Invalid build_options: [\'-c https://staging.continuum.io/prefect/llvm-21.1.8-stage2-build-0\']. Allowed: [\'--debug\', \'--error-overdepending\', \'--error-overlinking\', \'--no-error-overlinking\', \'--no-include-recipe\', \'--no-test\', \'--no-verify\', \'--quiet\', \'--skip-existing\', \'--suppress-variables\']","input":["--suppress-variables","--error-overlinking","-c https://staging.continuum.io/prefect/llvm-21.1.8-stage2-build-0"],"ctx":{"error":{}}},{"type":"value_error","loc":["body","tasks",2,"build","build_options"],"msg":"Value error, Invalid build_options: [\'-c https://staging.continuum.io/prefect/llvm-21.1.8-stage2-build-0\']. Allowed: [\'--debug\', \'--error-overdepending\', \'--error-overlinking\', \'--no-error-overlinking\', \'--no-include-recipe\', \'--no-test\', \'--no-verify\', \'--quiet\', \'--skip-existing\', \'--suppress-variables\']","input":["--suppress-variables","--error-overlinking","-c https://staging.continuum.io/prefect/llvm-21.1.8-stage2-build-0"],"ctx":{"error":{}}}]}'
Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/bin/pbp", line 10, in <module>
    sys.exit(cli())
             ~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1442, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1363, in main
    rv = self.invoke(ctx)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1226, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 794, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/graph.py", line 176, in create
    response = create_graph(host, graph, with_linter=with_linter, release=release)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/utils/graph_utils.py", line 185, in create_graph
    raise Exception("Failed to create graph")
Exception: Failed to create graph
'
aws lambda request id: ecd51e98-b3e6-5ca8-8600-7f4381dc98cb
[View in Grafana](https://grafana.anacondaconnect.com/explore?schemaVersion=1&panes=%7B%2286g%22:%7B%22datasource%22:%22loki-central%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7B__aws_cloudwatch_log_group%3D%5C%22%2Faws%2Flambda%2Fpbp-graph-submitter-prod%5C%22%7D%20%7C%20json%20%7C%20requestId%20%3D%20%60ecd51e98-b3e6-5ca8-8600-7f4381dc98cb%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki-central%22%7D,%22editorMode%22:%22builder%22,%22direction%22:%22backward%22%7D%5D,%22range%22:%7B%22from%22:%22now-3h%22,%22to%22:%22now%22%7D,%22panelsState%22:%7B%22logs%22:%7B%22columns%22:%7B%220%22:%22Time%22,%221%22:%22Line%22%7D,%22visualisationType%22:%22table%22,%22labelFieldName%22:%22labels%22%7D%7D%7D%7D&orgId=1)</details>

--- a/abs.yaml
+++ b/abs.yaml
@@ -2,6 +2,10 @@
 build_parameters:
   - "--suppress-variables"
   - "--error-overlinking"
-  - "-c https://staging.continuum.io/prefect/llvm-21.1.8-stage2-build-0"
+
+staging_channel_upload_target: llvm-21.1.8-stage2-build-0
+
+channels:
+  - https://staging.continuum.io/prefect/llvm-21.1.8-stage2-build-0
 
 aggregate_check: false

--- a/abs.yaml
+++ b/abs.yaml
@@ -6,6 +6,6 @@ build_parameters:
 staging_channel_upload_target: llvm-21.1.8-stage2-build-0
 
 channels:
-  - https://staging.continuum.io/prefect/llvm-21.1.8-stage2-build-0
+  - https://staging.continuum.io/pbp/llvm-21.1.8-stage2-build-0
 
 aggregate_check: false

